src/generic/toast-context/index.tsx

```diff
@@ -10,7 +10,8 @@ export interface ToastActionData {
 export interface ToastContextData {
   toastMessage: string | null;
   toastAction?: ToastActionData;
-  showToast: (message: string, action?: ToastActionData) => void;
+  capitilize?: boolean;
```

Comment:
What is the purpose of this new parameter? I don't think "capitalization" should be a feature of our toasts. Do we really need it?

Also, the spelling is wrong in several places. It should be "Capitalize" with two As.

Reply:
Removed here https://github.com/openedx/frontend-app-authoring/pull/1543/commits/bb90f0bb4f22b16135fb36dbcd1184565e01517d